apps/files/src/default.js

```diff
@@ -26,6 +26,9 @@ const appInfo = {
     }, {
       app: 'files-sharing',
       component: FileSharingSidebar,
+      enabled (capabilities) {
+        return capabilities.files_sharing.api_enabled === true
```

Comment:
Getting error "Cannot read property 'files_sharing' of undefined"

Reply:
did you log-off and logon again? capabilities are only loaded on login - not nice - but should work for now

Reply:
I'm getting it immediately after login. Without enabling/disabling shares.